apps/emqx_ft/src/emqx_ft_storage.erl

```diff
@@ -63,12 +71,53 @@ assemble(Transfer, Callback) ->
     Mod = mod(),
     Mod:assemble(storage(), Transfer, Callback).
 
+%%--------------------------------------------------------------------
+%% Local FS API
+%%--------------------------------------------------------------------
+
+-type filefrag() :: emqx_ft_storage_fs:filefrag().
+-type transferinfo() :: emqx_ft_storage_fs:transferinfo().
+
+-spec list_local(transfer(), fragment | result) ->
```

Comment:
🤔 shouldn't these function belong to `emqx_ft_storage_fs`?

other storages (like s3) may not have a concept of locality...